tools/topology/topology1/sof-tgl-sdw-max98373-rt5682.m4

```diff
@@ -67,9 +67,9 @@ ifelse(PLATFORM, `adl',
 	include(`platform/intel/intel-generic-bt.m4')
 ', `')
 #define BE dai_link ID
-define(`SMART_BE_ID', 2)
+define(`SMART_BE_ID', 0)
```

Comment:
@yongzhi1 We can keep the BE id unchanged as long as we define headset codec first in `struct snd_soc_acpi_link_adr`. For example
```
+static const struct snd_soc_acpi_link_adr adl_erb[] = {
+	{
+		.mask = BIT(2),
+		.num_adr = ARRAY_SIZE(rt5682_2_adr),
+		.adr_d = rt5682_2_adr,
+	},
+	{
+		.mask = BIT(0),
+		.num_adr = ARRAY_SIZE(mx8373_0_adr),
+		.adr_d = mx8373_0_adr,
+	},
+	{}
+};
```